docs/pages/includes/role-spec.mdx

```diff
@@ -25,7 +25,7 @@ spec:
     # certificates expire in the middle of an active session.
     # It overrides the global cluster setting.
     disconnect_expired_cert: no
-    # max_sessions is total number of session channels that can be established
+    # enterprise-only: max_sessions is the total number of session channels that can be established
```

Comment:
How about to match the rest of the changes.
```suggestion
    # the total number of session channels that can be established (enterprise-only)
```

Reply:
Will do!